<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmNavProgressBar</class>
 <widget class="QWidget" name="frmNavProgressBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="NavProgressBar" name="widget1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NavProgressBar" name="widget2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>2</verstretch>
      </sizepolicy>
     </property>
     <property name="currentBackground">
      <color>
       <red>24</red>
       <green>189</green>
       <blue>155</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NavProgressBar" name="widget3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="navStyle">
      <enum>NavProgressBar::NavStyle_TB</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NavProgressBar" name="widget4">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="navStyle">
      <enum>NavProgressBar::NavStyle_TB</enum>
     </property>
     <property name="currentBackground">
      <color>
       <red>24</red>
       <green>189</green>
       <blue>155</blue>
      </color>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NavProgressBar" name="widget5">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="navStyle">
      <enum>NavProgressBar::NavStyle_ZFB</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="NavProgressBar" name="widget6">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="navStyle">
      <enum>NavProgressBar::NavStyle_ZFB</enum>
     </property>
     <property name="currentBackground">
      <color>
       <red>24</red>
       <green>189</green>
       <blue>155</blue>
      </color>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavProgressBar</class>
   <extends>QWidget</extends>
   <header>navprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
